### ISSUE TITLE
School/Organization Field

### ISSUE THREAD

**Issue by dani-ji** [ji-devs/ji-cloud#814]
Add a field in the second step of registration for the user to either select from a dropdown of known schools (populated by a one time import) or input a new school/organization (when the user adds a new one it should not be added to the dropdown list)

There will currently be no backend functionality utilizing the school.oranization list and will be a simple text string on the user profile which will be accessible by the admin only via direct database query

@corinnewo please review this to confirm it aligns with your vision

**Comment by corinnewo** [comment]
Yes. It is what we discussed, Would it be possible to get a list of the extra, added entries once in a while (like once a fortnight) so I can add them to Salesforce? @dani-ji

**Comment by corinnewo** [comment]
@MendyBerger We need to add this to the backlog for LB. Will become relevant as soon as we have MVP and people are registering - need to be able to standardise the names of the organisations, and use names we already have in our CRM.

**Comment by MendyBerger** [comment]
Not sure I understand the exact requirements here, would like to discuss

**Comment by corinnewo** [comment]
[report1640333692155.csv](https://github.com/ji-devs/ji-cloud/files/7773403/report1640333692155.csv)
[Schools and Organizations-2021-12-24-10-14-32.xlsx](https://github.com/ji-devs/ji-cloud/files/7773404/Schools.and.Organizations-2021-12-24-10-14-32.xlsx)

**Comment by corinnewo** [comment]
@MendyBerger Thanks for your comment - please see the lists I have sent you in your email!
